polar2grid/readers/viirs_edr.py

```diff
@@ -149,4 +149,26 @@ def add_reader_argument_groups(
     if group is None:
         group = parser.add_argument_group(title="VIIRS EDR Reader")
 
+    group.add_argument(
+        "--filter-veg",
+        action=BooleanOptionalAction,
+        default=True,
+        help="Filter vegetation index variables by various quality flags. Default is enabled.",
+    )
+    group.add_argument(
+        "--aod-qc-filter",
+        default=1,
```

Comment:
Default is 1 like you mentioned in the related issue @kathys so that should include medium and high quality data. If that sounds good to you then I'll merge this. Oh...except I'll need to make sure the Satpy PR is merged first.